refinery/ui/source/js/dashboard/controllers/tutorial-launchpad.js

```diff
@@ -14,17 +11,17 @@ function launchpadTutorialCtrl ($scope, updateUserTutorials, tutorialPageNavigat
   };
 
   $scope.launchpadCompletedEvent = function () {
-    document.getElementById('launchpadStep1Fix').style['min-width'] = null;
+    document.getElementById('data-set-panel').style['min-width'] = null;
```

Comment:
This has been done on purpose since HTML only allows one ID. In my mind, the DOM element should not be given an ID that only makes sense for exactly one purpose (here for example the launch pad intro js tutorial step 1 fix). Because other parts of the code start to make less sense then. For example, when you go into CSS and find a rule related to `launchpadStep1Fix` you have no idea what kind of element that is. Also, in other tutorials (and this is the actual reason why I renamed it in the first place) it starts to make even less sense. A satori tutorial uses the same element as step6 for example. So in general I would advocate to use generic ID names that somehow tell the developer what kind of element this is so it sort of makes sense for several use cases.